release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -65,6 +65,16 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 * With this update, you can export your application in the ZIP file format to another project or cluster using the *Export application* option on the *+Add* page. 
 * With this update, you can create a Kafka event sink to receive events from a particular source and send them to a Kafka topic.
 
+[id="ocp-4-12-helm-page-improvements"]
+==== Helm page improvements
+
+In {product-title} 4.12, you can use the Helm page to:
+
+* Create Helm Releases and Repositories using the *Create* button.
+* Create, update, or delete a cluster-scoped or a namespace-scoped Helm chart repository.
+* View the list of the existing Helm chart repositories with their scope in the *Repositories* tab.
```

Comment:
Would you verify that there is really a *Repositories* tab? I'm not seeing any Helm-related instances of that term in our openshift-docs repo. The closest mention I see is `* Use the *Chart Repositories* filter to filter charts provided by *Red Hat Certification Charts* or *OpenShift Helm Charts*.` in `modules/helm-filtering-helm-charts-by-certification-level.adoc`

Reply:
It seems like our docs lean toward using "page" rather than "tab".

Reply:
I had added the term page earlier but was corrected by the QE to change it to tab. I am adding the PR link here (https://github.com/openshift/console/pull/11844) requesting for additional clarity from you. Since this is a new tab, I suspect that is the reason why we may have not seen such instances before in our documentation.

Reply:
WRT new `*Helm* page`, that makes sense.

Reply:
I had seen the SME's comment requesting "tab," too. In this case, we are referring to the page that the user sees after they click the tab.

I just looked it up to get more clarity on our usage. In "Usage and highlighting for interface elements" section of the IBM Style Guide it says:

* Tab = "An interface element that is used to
organize distinct sets of information
and to provide an easy way to navigate
between these sets. Tabs are used on
websites, in applications, and in
notebooks.
The control that you click to show the
information is always called a tab, and
the name of the area that contains the
information can vary depending on the
interface: it might be a page, pane,
view, or tab."
* Page = "In a notebook or wizard, a set of
interface elements that is organized to
help users accomplish tasks; on the
web, any document that can be
accessed by a uniform resource locator
(URL)."

Reply:
Fixed.